### PR TITLE
Add new boefjes and normalizers cover images to KAT-alogus

### PR DESCRIPTION
**Pull request by Rieven**
### Changes
Add new boefjes and normalizers cover images to KAT-alogus

### Issue link
https://github.com/minvws/nl-kat-coordination/issues/2853

Closes https://github.com/minvws/nl-kat-coordination/issues/2853

### Demo
image

image

image

image

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified.
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [x] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [x] I have made corresponding changes to the documentation, if necessary.
- [x] I have included comments in the code to elaborate on what is not self-evident from the code itself, including references to issues and discussions online, or implicit behavior of an interface.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: boefjes/boefjes/katalogus/local_repository.py [hunk 1]
```diff
@@ -62,20 +62,25 @@ def schema(self, id_: str) -> dict | None:
...
+        normalizers = self.resolve_normalizers()
+        default_cover_path = self.default_cover_path()
 
-        if id_ not in boefjes:
-            return self.default_cover_path()
-
-        boefje = boefjes[id_]
-        path = boefje.path / "cover.jpg"
+        if id_ not in boefjes and id_ not in normalizers:
+            return default_cover_path
```
Comment (ammar92) [inline boefjes/boefjes/katalogus/local_repository.py:69]: You might as well move this to an else-clause in the if-clause below

File: boefjes/boefjes/katalogus/local_repository.py [hunk 2]
```diff
@@ -62,20 +62,25 @@ def schema(self, id_: str) -> dict | None:
...
 
-        if not path.exists():
-            logger.debug("Did not find cover for boefje %s", boefje)
-            return self.default_cover_path()
+        if id_ in boefjes:
+            plugin = boefjes[id_]
+            cover_path = plugin.path / "cover.jpg"
+        elif id_ in normalizers:
+            plugin = normalizers[id_]
+            cover_path = plugin.path / "normalizer_cover.jpg"
```
Comment (ammar92) [inline boefjes/boefjes/katalogus/local_repository.py:76]:
```suggestion
            cover_path = plugin.path / "normalizer_cover.jpg"
        else:
            cover_path = default_cover_path
```

File: boefjes/boefjes/katalogus/local_repository.py [hunk 3]
```diff
@@ -62,20 +62,25 @@ def schema(self, id_: str) -> dict | None:
...
 
-        logger.debug("Found cover for boefje %s", boefje)
+        if not cover_path.exists():
+            logger.debug("Did not find cover for boefje %s", plugin)
+            return default_cover_path
 
-        return path
+        logger.debug("Found cover for boefje %s", plugin)
```
Comment (ammar92) [inline boefjes/boefjes/katalogus/local_repository.py:82]: I think we can remove the debug line, it doesn't make sense anymore. The logs about missing files are more helpful.

**Comment by Rieven** [comment]
> Just a few remarks. Have you also compressed/ minified the images?

Yes

File: boefjes/boefjes/katalogus/local_repository.py (hunk 1 above)
Comment (Rieven) [reply boefjes/boefjes/katalogus/local_repository.py]: I have removed this as well, will be indeed handled in the else clasue

**Review by ammar92 — APPROVED** [review]
No more remarks

**Comment by stephanie0x00** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
Looks good! All required new KAT-pictures are now added. Most pictures are also nicely centered, those which are not will be picked up in a different ticket (#2871). 

### What doesn't work:
n/a

### Bug or feature?:
n/a

**Review by underdarknl — APPROVED** [review]

**Pull request merged**